samples/nrf_desktop/src/services/hids.c

```diff
@@ -168,9 +168,14 @@ static void feature_report_handler(struct bt_gatt_hids_rep const *rep,
 
 		struct config_event *event = new_config_event();
 
-		event->id = rep->data[0];
-		memcpy(event->data, &(rep->data[1]), sizeof(event->data));
-		event->store_needed = true;
+		memcpy(&event->recipient, &(rep->data[0]), sizeof(event->recipient));
+		if (event->recipient != CONFIG_BT_GATT_DIS_PNP_PID) {
+			LOG_WRN("Drop event addressed to %x", event->recipient);
+			k_free(event);
```

Comment:
please do not allocate a memory that would be freed few lines later without being needed (you can check rep->data to see recipient)

Reply:
Done, the same done in USB